engine/intro_menu.asm

```diff
@@ -1035,7 +1035,7 @@ StartTitleScreen: ; 6219
 	ld [rSVBK], a
 
 	ld hl, rLCDC
-	res 2, [hl]
+	res rLCDC_SPRITE_SIZE, [hl] ; 8x8
```

Comment:
The 8x8/8x16 meaning would also be useful next to the `rLCDC_SPRITE_SIZE` definition.